pkg/virt-config/virt-config.go

```diff
@@ -244,6 +249,11 @@ func (c *ClusterConfig) GetSupportedAgentVersions() []string {
 }
 
 func (c *ClusterConfig) GetOVMFPath(arch string) string {
+	oldOvmfPath := c.GetConfig().OVMFPath
+	if oldOvmfPath != "" {
+		return oldOvmfPath
+	}
+
```

Comment:
shouldn't we do the precedence to the new `ArchitectureConfiguration`?
```
	switch arch {
	case "arm64":
		return c.GetConfig().ArchitectureConfiguration.Arm64.OVMFPath
	case "ppc64le":
		return c.GetConfig().ArchitectureConfiguration.Ppc64le.OVMFPath
	case "amd64":
		return c.GetConfig().ArchitectureConfiguration.Amd64.OVMFPath
	default:
	    oldOvmfPath := c.GetConfig().OVMFPath
	    if oldOvmfPath != "" {
		    return oldOvmfPath
	    }
		return c.GetConfig().ArchitectureConfiguration.Amd64.OVMFPath
	}
```

Reply:
@acardace @fossedihelm same here

Reply:
It means that if the user already opted-in for `ArchitectureConfiguration` with this change we will return cluster default